code/model/SiteTreeFileExtension.php

```diff
@@ -10,14 +10,43 @@ class SiteTreeFileExtension extends DataExtension {
 	);
 
 	public function updateCMSFields(FieldList $fields) {
-		$fields->insertAfter(new ReadonlyField('BackLinkCount', 
-			_t('AssetTableField.BACKLINKCOUNT', 'Used on:'), 
-			$this->BackLinkTracking()->Count() . ' ' . _t('AssetTableField.PAGES', 'page(s)')), 
+		$fields->insertAfter(
+			ReadonlyField::create(
+				'BackLinkCount', 
+				_t('AssetTableField.BACKLINKCOUNT', 'Used on:'), 
+				$this->BackLinkTracking()->Count() . ' ' . _t('AssetTableField.PAGES', 'page(s)'))
+			->addExtraClass('cms-description-toggle')
+			->setDescription($this->BackLinkHTMLList()),
 			'LastEdited'
 		);
 	}
 
 	/**
+	 * Generate an HTML list which provides links to where a file is used.
+	 *
+	 * @return String
+	 */
+	public function BackLinkHTMLList() {
+		$html = '<em>This list shows all pages where the file has been added through a WYSIWYG editor.</em>';
+
+		$html .= '<ul>';
+
+		foreach ($this->BackLinkTracking() as $backLink) {
+			$listItem = '<li>';
+
+			// Add the page link
+			$listItem .= '<a href="' . $backLink->Link() . '" target="_blank">' . $backLink->MenuTitle . '</a> &ndash; ';
```

Comment:
You need to XML encode this title, otherwise you could get HTML entities in your text.